fix the conflict behavior (#2635)
risingwavelabs/risingwave-docs: docs/sql/commands/sql-create-table.md

When `DO UPDATE IF NOT NULL` behavior is applied, `DEFAULT` clause is not allowed on the table's columns.
:::

:::note

> In version "1.9", "1.10" and "2.0", `DO NOTHING` and `DO UPDATE FULL` were not supported. On these versions, `DO NOTHING` could be replaced with `IGNORE`, and `DO UPDATE FULL` could be replaced with `OVERWRITE`. In subsequent versions, the syntax for `IGNORE` and `OVERWRITE` will still be supported.

Reviewer: ```suggestion
In versions 1.9, 1.10, and 2.0, `DO NOTHING` and `DO UPDATE FULL` were not supported. `DO NOTHING` was replaced by `IGNORE`, and `DO UPDATE FULL` was replaced by `OVERWRITE`. In subsequent versions, the syntax for `IGNORE` and `OVERWRITE` remains supported.
```

> :::

Author: Need help to check if the this sentence is appropriate. 🙏 @WanYixian
